sources/functions/jellyfin

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Contributers: katiethedev
+# This functions file is created to resolve some replicated code between the install and update functions.
+
+# This function adds the Jellyfin repository.
+function add_jellyfin_repo() {
+    # Set required variables
+    ARCHITECTURE="$(_os_arch)"
+    BASE_OS="$(_os_distro)"
+    case "${BASE_OS}" in
+        raspbian)
```

Comment:
I don't think raspbian will pass [this](https://github.com/swizzin/swizzin/blob/develop/setup.sh#L197) check so we don't need to write a case for it